From fca08ca2076b59e4de7ba37f3f53e55c5ec46326 Mon Sep 17 00:00:00 2001
From: Ryan Rector <rmrector@gmail.com>
Date: Thu, 21 Feb 2019 17:54:25 -0700
Subject: [PATCH] Set NFO thumb without aspect as poster

Small adjustment to #12612 to keep backward-compatibility with
NFOs exported from Kodi 11 Eden scrapers.
---
 xbmc/video/VideoInfoScanner.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xbmc/video/VideoInfoScanner.cpp b/xbmc/video/VideoInfoScanner.cpp
index 49e9b1899e25..b58963480e9a 100644
--- a/xbmc/video/VideoInfoScanner.cpp
+++ b/xbmc/video/VideoInfoScanner.cpp
@@ -1504,8 +1504,8 @@ namespace VIDEO
         continue;
       std::string aspect = url.m_aspect;
       if (aspect.empty())
-        // temporary support for XML music video scrapers that share music album scraper bits
-        aspect = content == CONTENT_MUSICVIDEOS ? "poster" : "thumb";
+        // Backward compatibility with Kodi 11 Eden NFO files
+        aspect = ContentToMediaType(content, pItem->m_bIsFolder) == MediaTypeEpisode ? "thumb" : "poster";
       if (find(artTypes.begin(), artTypes.end(), aspect) == artTypes.end() || art.find(aspect) != art.end())
         continue;
       std::string image = GetImage(url, pItem->GetPath());
